description_templates/golden/postgres:vshn.txt

```diff
@@ -0,0 +1,4 @@
+VSHN-managed PostgreSQL
```

Comment:
It's actually called "PostgreSQL by VSHN", see https://products.docs.vshn.ch/products/appcat/postgresql.html